test.py

```diff
@@ -14,6 +14,7 @@
 import html
 import itertools
 import sys
+from typing import cast
 
 import widlparser
```

Comment:
```suggestion
from widlparser import Construct
```

And then you can drop 'widlparser.' from the casts.